compiler/test/stdlib/lists.test.gr

```diff
@@ -24,8 +24,6 @@ assert append([], [4]) == [4]
 
 # List.contains
 
-assert contains(true, list) == false
-assert contains(false, list) == false
```

Comment:
Since I merged the `==` change, these should be type errors now. That breaks on the GitHub actions CI, but it didn't on Travis... odd.